requirements.python.txt

```
@@ -2,7 +2,7 @@
 setuptools
 
 # Ansible
-ansible-core==2.16.3
+ansible-core==2.15.6
```

```suggestion
ansible-core>=2.15.6
```
Can this be modified to the above?

---

Yes done